Makefile

```diff
@@ -1,20 +1,32 @@
-MAKE=make
-LD=ld
-RM=rm
+include mk/build.mk
 
-all: boot-subsystem kernel-subsystem
+BOOTIMAGE=hda.img
 
+all: bootimage
```

Comment:
Could the creation of a bootable image be moved to a separate target so that it's not part of all?  It's completely a personal preference on my part, but I prefer to keep the disk images and whatnot in a separate tree because to me they're conceptually different parts.  Maybe something like make all-bootimage (or ab to save typing?) so that those who prefer can have a straightforward way to build the image within the build tree, while those who don't don't have it put there.

This also would make it easier for people who want to experiment with different disk sizes (probably not an issue at the moment, but as it grows) to do so without having their changes overwritten every time they pull from the repo (there are other practical issues I can see emerging if the project moves along, with overwriting data, etc.).

Reply:
Yup, no problem.

Reply:
Done.  Make all now just makes the bin files